.mergify.yml

```diff
@@ -12,7 +12,7 @@ pull_request_rules:
 
   - name: auto label PRs from reviewers
     conditions:
-      - author=@reviewers
+      - author=@cryostatio/reviewers
```

Comment:
I don't think the Mergify docs are particularly clear or consistent about this:

https://docs.mergify.com/conditions/#attributes

https://docs.mergify.com/actions/request_reviews/#flexible-reviewers-assignment

It seems like in the `actions: foo: teams:` attribute you must specify the whole `@org/teamname`, but in `conditions:` it is just `author=@teamname`

Reply:
Ahh right! Updated now :))